components/file_system/src/iosnoop/null.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
+
+use super::{IOStats, IOType};
+use std::cell::Cell;
+
+pub struct IOContext;
+
+impl IOContext {
+    pub fn new() -> Self {
+        IOContext {}
+    }
+
+    #[allow(dead_code)]
+    fn delta(&self) -> IOStats {
+        IOStats::default()
+    }
+
+    #[allow(dead_code)]
+    fn delta_and_refresh(&mut self) -> IOStats {
+        IOStats::default()
+    }
+}
+
+pub fn init_io_snooper() -> Result<(), String> {
```

Comment:
do we need to define a error type for the `file_sytem` crate and integrate with the `error_code` crate? (we can do that in another PR)

Reply:
Seems no need, only need to print the error message anyway when bootstrapping TiKV.